crates/rpc/rpc-eth-api/src/helpers/call.rs

```diff
@@ -667,7 +679,7 @@ pub trait Call:
         for tx in transactions {
             if *tx.tx_hash() == target_tx_hash {
                 // reached the target transaction
-                break
+                break;
```

Comment:
please undo all additional `;`

Reply:
@mattsse resolved (my ide formats on saving the file, undone)